README.md

```diff
@@ -1,7 +1,5 @@
 # tedious-connection-pool
-[![Dependency Status](https://david-dm.org/tediousjs/tedious-connection-pool.svg)](https://david-dm.org/tediousjs/tedious-connection-pool)
-[![npm version](https://badge.fury.io/js/tedious-connection-pool.svg)](https://badge.fury.io/js/tedious-connection-pool)
-[![Build status](https://ci.appveyor.com/api/projects/status/jnurb48ao1wrbgbr?svg=true)](https://ci.appveyor.com/project/ben-page/tedious-connection-pool)
+npm version: 1.0.6
```

Comment:
@jschell12 as much as I like getting rid of those stupid badges, I suppose we ought to call out we're doing that in the PR description if we're going to do it